lib/sky-pages-module-generator.js

```diff
@@ -83,6 +103,12 @@ function getSource(skyAppConfig) {
     'AppExtrasModule'
   ];
 
+  if (skyAppConfig.skyux.help) {
+    nodeModulesImports.push(`import { BBHelpModule } from '@blackbaud/skyux-lib-help';`);
```

Comment:
I'm nitpicking, but can you name this `nodeModuleImports` so the case matches `runtimeModuleImports`?